app/test_main.py

```diff
@@ -1,3 +1,38 @@
+import pytest
+
 from app.main import get_human_age
 
-# write your code here
+
+@pytest.mark.parametrize(
+    "cat_age,dog_age,result",
+    [
+        pytest.param(0, 0, [0, 0], id="animals' age is 0"),
+        pytest.param(0, 14, [0, 0], id="cat's age is less than 15"),
+        pytest.param(14, 0, [0, 0], id="dog's age is less than 15"),
+        pytest.param(15, 0, [1, 0], id="dog's age is 15"),
+        pytest.param(0, 15, [0, 1], id="cat's age is 15"),
+        pytest.param(
+            16, 0, [1, 0], id="cat's age is more than 15 less than 24"
+        ),
+        pytest.param(
+            0, 23, [0, 1], id="dog's age is more than 15 less than 24"
+        ),
+        pytest.param(24, 23, [2, 1], id="cat's age is 24"),
+        pytest.param(15, 24, [1, 2], id="dog's age is 24"),
+        pytest.param(28, 23, [3, 1], id="cat's age is more than 24"),
+        pytest.param(15, 29, [1, 3], id="dog's age is more than 24"),
```

Comment:
what about big/negative nums ?

Reply:
@YuriiKindrat why big numbers are different from any number more than 24?

Reply:
> Add more tests, for all conditions from ur readme.md file and checklist

I'm literally trying to follow the very first rule in the checklist - Make sure that your tests don't test the same case few times.
i got it about why we have to check cat's and dog's age separately. because an implementation of the function might be specific for each animal.
negative nums are strange things to pass to the function, but I'll add the test. 
but it seems that 100 and 30 must work the same, using the same logic at the same "rule range"